src/pool/srv_pool.c

```diff
@@ -4905,6 +4905,7 @@ ds_pool_upgrade_if_needed(uuid_t pool_uuid, struct rsvc_hint *po_hint,
 	int				rc;
 	bool				schedule_layout_upgrade = false;
 	bool				need_put_leader = false;
+	bool				schedule_upgrade = false;
```

Comment:
code readability suggestion (not a showstopper I guess):

rename schedule_upgrade -> request_schedule_upgrade (or want_schedule_upgrade)
rename schedule_layout_upgrade -> scheduled_layout_upgrade